hack/verify-codecgen.sh

```diff
@@ -102,6 +104,8 @@ for current in ${index[@]}; do
   sed 's/YEAR/2015/' "${initial_dir}/hack/boilerplate/boilerplate.go.txt" > "${base_generated_file}.tmp"
   cat "${base_generated_file}.1tmp" >> "${base_generated_file}.tmp"
   rm "${base_generated_file}.1tmp"
+  # restore the generated file
```

Comment:
codecgen now pays attention to whether a custom encode/decode func has already been implemented for the object. update-codecgen.sh removes the generated file and regenerates it, so it is starting with a clean slate. verify-codecgen.sh generates to a new tmp file, and the existing real file was causing it to behave differently. My change moves the real file to a .bak file so it doesn't affect the verify copy, then moves it back for diffing.